src/openassetio-core/src/hostApi/ManagerFactory.cpp

```diff
@@ -16,6 +17,11 @@
 #include <openassetio/typedefs.hpp>
 #include "openassetio/InfoDictionary.hpp"
 
+namespace {
+constexpr std::string_view kConfigDirVarNonBraced = "$config_dir";
+constexpr std::string_view kConfigDirVarBraced = "${config_dir}";
```

Comment:
[Consider] If it's as simple as just adding 2 more constexpr's, I'd argue for adding the capitalised versions of these too, it can be frustrating when you want to keep to your style and can't.